### ISSUE TITLE
Events page screen reader accessibility fixes

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#137]
## Description

There's a couple of tricky a11y issues for this page, so I'll do my best to break them down in one mega ticket. Feel free to split this up! I'll add some success criteria at the bottom to be specific.

## Headings

Right now, the filter's `h3`s and the event `h3`s are combined together, which isn't cash money.

We'll want to reorder the headings like this to give that nice logical hierarchy

```
h1 - Events
  h2 - Filters
    h3 - Date
    h3 - etc
  h2 - Showing 1 to 12 of 12 events
    h3 - Event name
    h3 - etc
  h2 - FAQ
  etc
```
### Success criteria for headings

- [x] "2022 Events" should be renamed to "Filters"
- [x] "x to y of z" should be a `h2`
- [x] If possible, append a static "Showing" at the beginning and "Events" to the end of it? e.g. "Showing 1 to 12 of 12 events"

## Filter legends

With our heading fixes, we can remove the `aria-label` from our legends, otherwise they'll start getting a bit repetitive. We can rely on the button text to do the trick here as it's nested within the `legend`

### Success criteria for legends

- [x] Delete `aria-label` for legends

## Live regions 🎤 

Last piece of the puzzle is to drop our "x to y out of z" friend within a live region.

By adding `aria-live="polite"` to this `div` (or I think any of its parents) we'll get a nice polite announcement every time a change is made 😄 (but without the twice music, pls ignore that or Silvia will roast me)

`<div class="mixitup-page-stats" aria-live="polite">1 to 5 of 5</div>`

https://user-images.githubusercontent.com/14154792/192180781-3f527697-c573-4514-a2e9-e1f8b51db6ca.mov

### Success criteria for live regions

- [x] Add `aria-live="polite"` to the container of "x to y out of z"
